apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -98,14 +99,11 @@ export default class AnimationPickerListItem extends React.Component {
                 {label}
               </span>
             )}
-            {category && iconImageSrc && (
-              // TODO: A11y279 (https://codedotorg.atlassian.net/browse/A11Y-279)
-              // Verify or update this alt-text as necessary
+            {iconImageSrc && (
               <img
-                data-category={category}
                 className={style.categoryImage}
                 src={iconImageSrc}
-                alt=""
+                alt={''}
```

Comment:
I think no alt text is appropriate here because the button that this `img` is in has an appropriate `aria-label` with the label for each categeory (eg, "Animals").